src/main/java/rpgram/RPGram.java

```diff
@@ -211,6 +218,8 @@ public void onUpdateReceived(Update update) {
                     treasure = new Treasure(curPlayer.getInventory().get(id), curPlayer.getPos(), curPlayer.getName());
```

Comment:
It's a good idea to keep lines no more than 80 characters long.

Reply:
`Treasure` logic should be rewritten anyway, but thanks for the tip.